app/components/blacklight/system/flash_message_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="alert <%= @classes %>">
   <%= message %>
-  <%= tag.button_tag button_contents, type: "button", class: "btn-close",
+  <%= button_tag button_contents, type: "button", class: "btn-close",
```

Comment:
```suggestion
  <%= tag.button button_contents, type: "button", class: "btn-close",
```

Was maybe the intent?

Reply:
Thanks @cbeer -- not sure the difference, other than `button_tag` seems to generate an additional `name` attribute. Happy to switch to this though.

<img width="1020" alt="Screenshot 2024-05-10 at 17 19 49" src="https://github.com/projectblacklight/blacklight/assets/1328900/9f9d0de0-9372-42ab-abc1-fe30c35304a2">